octopoes/tests/integration/test_unicode.py

```diff
@@ -0,0 +1,85 @@
+import os
+import time
+import uuid
+from datetime import datetime
+
+import pytest
+
+from octopoes.api.models import Declaration, Observation
+from octopoes.connector.octopoes import OctopoesAPIConnector
+from octopoes.models import DeclaredScanProfile, ScanLevel
+from octopoes.models.ooi.dns.zone import Hostname
+from octopoes.models.ooi.network import Network
+from octopoes.models.origin import OriginType
+
+if os.environ.get("CI") != "1":
+    pytest.skip("Needs XTDB multinode container.", allow_module_level=True)
```

Comment:
This is no longer necessary AFAICS, because if you do "make itest" the tests will be run with the XTDB container and CI is always set to 1.

Reply:
Try removing it and do a `make -C octopoes test`.

Reply:
Ok I thought we ran the integration tests here the same way we as we do in rocky. It's ok for now and maybe something we should fix in a different PR.